container/raw/handler.go

```diff
@@ -180,6 +182,20 @@ func (self *rawContainerHandler) GetSpec() (info.ContainerSpec, error) {
 
 	// The raw driver assumes unified hierarchy containers.
 
+	// Get the lowest creation time from all hierarchies as the container creation time.
+	now := time.Now()
+	lowestTime := now
+	for _, cgroupPath := range self.cgroupPaths {
+		// The modified time of the cgroup directory is when the container was created.
+		fi, err := os.Stat(cgroupPath)
+		if err == nil && fi.ModTime().Before(lowestTime) {
+			lowestTime = fi.ModTime()
```

Comment:
Would this work for things like systemd containers that just join cpu and names hierarchies? Won't it always pick the machine boot time in that case?